rootfs/usr/local/bin/tailscale_ctl

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+start_tailscale() {
+	# Start tailscaled with userspace tunneling - required for running in a container
+	tailscaled -tun userspace-networking &
+
+	authkey="$(sops --decrypt --extract '["tailscale_oauth_client_secret"]' /mnt/workspace/source/config/secrets/global.yaml)"
```

Comment:
I'm not sure how flexible we should be here in terms of secrets path and key name. `spacelift-configure-paths` already contains hardcoded paths though, so this is probably fine.